src/util/redux.js

```diff
@@ -0,0 +1,246 @@
+import {
```

Comment:
if this is specific to `redux` then it belongs in it's own file, `state-management.js` is not specific to redux.